test/reducers/reducer-test.js

```diff
@@ -107,5 +115,71 @@ describe('todos reducer', () => {
          id: 0
        }
      ])
-   })
+   });
+
+  it('should handle TOGGLE_TODO', () => {
+    expect(
+        todos([
+          {
+            text: 'Run the tests',
+            completed: false,
+            id: 1
+          }, {
+            text: 'Use Redux',
+            completed: false,
+            id: 0
+          }
+        ], {
+          type: 'TOGGLE_TODO',
+          payload: {
+            id: 1
+          }
+        })
+    ).toEqual([
+      {
+        text: 'Run the tests',
+        completed: true,
+        id: 1
+      }, {
+        text: 'Use Redux',
+        completed: false,
+        id: 0
+      }
+    ])
+  });
+  //
+  //it('should handle MOVE_TODO', () => {
+  //  expect(
+  //      todos([
+  //        {
+  //          text: 'Run the tests',
+  //          completed: false,
+  //          id: 0
+  //        }, {
+  //          text: 'Use Redux',
+  //          completed: false,
+  //          id: 1
+  //        }
+  //      ], {
+  //        type: 'MOVE_TODO',
+  //        payload: {
+  //          id: 0,
+  //          text: 'Run the tests',
+  //          completed: false,
+  //          target_id: 1
+  //        }
+  //      })
+  //  ).toEqual([
+  //    {
+  //      text: 'Use Redux',
+  //      completed: false,
+  //      id: 1
+  //    },
+  //    {
+  //      text: 'Run the tests',
+  //      completed: false,
+  //      id: 0
+  //    }
+  //  ])
+  //})
```

Comment:
Yes i will. This test case was not being passed and was throwing some error. So i had commented it out. Sorry, should have removed it.